dietpi/preboot

```diff
@@ -232,9 +226,6 @@
 		#	Static IPs
 		if (( $use_static )); then
 
-			#enable dns-nameservers
-			sed -i 's/^#dns-nameservers/dns-nameservers/g' /etc/network/interfaces
```

Comment:
Needs testing/checking rest of code for enable, as this must be enabled for STATIC IP to function.